q2_moshpit/busco/busco.py

```diff
@@ -125,9 +128,20 @@ def _evaluate_busco(
     scaffold_composition: bool = False,
 ) -> pd.DataFrame:
     kwargs = {
-        k: v for k, v in locals().items() if k not in ["bins",]
+        k: v for k, v in locals().items() if k not in ["bins", "busco_db"]
     }
 
+    # Add busco_db to kwargs
+    if busco_db is not None:
+        kwargs["offline"] = True
+        kwargs["download_path"] = f"{str(busco_db)}/busco_downloads"
+
+    if lineage_dataset is not None:
+        _validate_lineage_dataset_input(
+            lineage_dataset, auto_lineage, auto_lineage_euk, auto_lineage_prok,
+            busco_db, kwargs  # this may be modifies inside this function
```

Comment:
You mean for kwargs right?
```suggestion
            busco_db, kwargs  # kwargs may be modified inside this function
```